tmt/cli.py

```diff
@@ -1666,6 +1666,11 @@ def try_command(context: Context, image_and_how: str, **kwargs: Any) -> None:
     \b
         tmt try fedora@container
         tmt try centos-stream-9@virtual
+
+    Or connect to the running guest by specifying FQDN or IP address:
```

Comment:
```suggestion
    Or connect to the running guest by specifying hostname or IP address:
```

Let's use rather `hostname` than the abbreviation?

Reply:
it's not the same, is it? https://serverfault.com/questions/269838/what-is-the-difference-between-a-hostname-and-a-fully-qualified-domain-name

Reply:
Agreed to keep FQDN on hacking session.